src/Operations.jl

```diff
@@ -1130,10 +1130,14 @@ function free(ctx::Context, pkgs::Vector{PackageSpec})
     end
 end
 
-function gen_test_code(testfile::String; coverage=false)
+function gen_test_code(testfile::String;
+        coverage=false,
+        julia_args::AbstractVector{<:AbstractString}=String[],
```

Comment:
Maybe this should be `AbstractVector{Cmd}`? And you specify it as
```
Pkg.test("Example"; julia_args = [`--startup-file=no`, `-O3`])
```
?

Reply:
I like it.

Reply:
Done.